### ISSUE TITLE
Why use probabilities P_I and P_T instead of  logits?

### ISSUE THREAD

**Issue by madoka109** [ChangyaoTian/VL-LTR#2]
And that's why your L_rec doesn't need to be divided by 2?

**Comment by ChangyaoTian** [comment]
Thank you for your comment. Here we just follow the standard implementation of CrossEntropy loss in Pytorch, please refer to the [code](https://github.com/ChangyaoTian/VL-LTR/blob/445a7e03c592bc6e1647962b29bb64e0cb658ec7/main.py#L365) directly，which means using the prob or the logits is nothing but a matter of expression.
As for not adding the additional division of 2 for L_rec, well, you can see it is just a matter of scale, isn't it?
